blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/AstyanaxReader.java

```diff
@@ -484,6 +486,16 @@ private MetricData transformColumnsToMetricData(Locator locator, ColumnList<Long
         }
     }
 
+    private DataType getDataType(Locator locator, String dataTypeCacheKey) throws CacheException{
+        String meta = metaCache.get(locator, dataTypeCacheKey);
+        DataType dataType = null;
+        if (meta != null) {
+            dataType = new DataType(meta);
+        }
+
+        return dataType;
```

Comment:
I'm kinda in favor of keeping the curly brackets: https://www.imperialviolet.org/2014/02/22/applebug.html